api/models/constants.py

```diff
@@ -1,2 +1,43 @@
 DEFAULT_CHAR_FIELD_MAX_LEN = 255
 DEFAULT_CHOICE_CHAR_FIELD_MAX_LENGTH = 15
+
+# IMPORTANT: the boundaries of larger ranges must match the boundaries of the smaller ones
+
+# Match string ranges that are presented to the user to actual ranges, e.g.:
+# {"6-8": (6, 9), "9-11": (9, 12), ...}
+STUDENT_AGE_RANGE_BOUNDARIES = {
+    str_range: (int(str_range.split("-")[0]), int(str_range.split("-")[1]) + 1)
```

Comment:
In my opinion, if you just will do it as you wrote in example, it will be more readable.

`STUDENT_AGE_RANGE_BOUNDARIES  = {"6-8": (6, 9), "9-11": (9, 12), ...}`

Reply:
OK. Just tried to avoid repetition